recipes/ruptures/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "ruptures" %}
+{% set version = "1.0.2" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 1c4ae07622c26672ef6a113f6ac75b8c36f75bb9e649bf1bbc2c9afa23aee881
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  noarch: python 
+requirements:
+  host:
+    - numpy
```

Comment:
Usually `pip` and `python` are the only dependencies needed in `host`, while the other can go to `run`. Can you please try that?